admin_interface/admin.py

```diff
@@ -19,10 +23,7 @@ class ThemeAdmin(admin.ModelAdmin):
             None,
             {
                 "classes": ("wide",),
-                "fields": (
-                    "name",
-                    "active",
-                ),
+                "fields": ("name", "active", "active_dark"),
```

Comment:
I think having values on multiple lines is better because adding/removing a field would impact only the line of that field.

Reply:
Wonder if it’s black that made this change because there was no trailing comma after the last list item.